### PR TITLE
Update interaction

### LINKED ISSUE #610: Interactive plots: Toogle time unit in time plots
## General

* pyfar version: 0.6.5

## Description

It would be nice to have a shortcut to toogle the time unit in time domain plots between seconds, milliseconds, microseconds, and samples. `shift + x` could do this, which is toogling between linear and logarithmic axis for frequency domain plots.

### PR DESCRIPTION
**Pull request by f-brinkmann**
### Which issue(s) are closed by this pull request?

Closes #610 

### Changes proposed in this pull request:

- [x] add toggling unit of time plots to interaction (#610)
- [x] add toggling of mode for time plots of complex signals
- [x] add toggling of side for frequency plots of complex signals
- [x] Add new shortcur to documentation
- [x] update tests

### CODE REVIEW

**Review by ahms5 — APPROVED** [review]

File: docs/resources/plot_shortcuts.rst [hunk 1]
```diff
@@ -78,7 +80,14 @@ Note that not all plots are available for TimeData and FrequencyData objects as
...
+
+  - linear and logarithmic axis scaling for frequency axes,
+  - seconds, milliseconds, microseconds, and samples for time axes,
+  - linear amplitude and amplitude in dB for axes showing amplitudes,
+  - wrapped and unwrapped phase for axes showing phase phase information.
+
+- Toggling the x-axis style is supported by: :py:func:`~pyfar.plot.time`, :py:func:`~pyfar.plot.freq`, :py:func:`~pyfar.plot.phase`, :py:func:`~pyfar.plot.group_delay`, :py:func:`~pyfar.plot.spectrogram`, :py:func:`~pyfar.plot.time_freq`, :py:func:`~pyfar.plot.freq_phase`, :py:func:`~pyfar.plot.freq_group_delay` (and their 2d versions)
+- Toggling the y-axis style is supported by: :py:func:`~pyfar.plot.time`, :py:func:`~pyfar.plot.freq`, :py:func:`~pyfar.plot.phase`, :py:func:`~pyfar.plot.group_delay`, :py:func:`~pyfar.plot.spectrogram`, :py:func:`~pyfar.plot.time_freq`, :py:func:`~pyfar.plot.freq_phase`, :py:func:`~pyfar.plot.freq_group_delay` (and their 2d versions)
+- Toggling the colormap style is supported by all 2d plots
 - Toggling between line and 2D plots is not supported by: spectrogram
```
Comment (sikersten) [inline docs/resources/plot_shortcuts.rst:93]: Maybe replace spectrogram with py:func:`~pyfar.plot.spectrogram` here?

File: pyfar/plot/_interaction.py [hunk 2]
```diff
@@ -333,13 +381,20 @@ def update(self, plot):
             self._x_values = ['log', 'linear']
             self._x_id = self._x_values.index(getattr(self, self._x_param))
             # y-axis
-            self._y_type = ['other', 'other', 'other', 'other', 'other']
+            self._y_type = ['other', 'other', 'other', 'other']
             self._y_param = 'unit_gd'
-            self._y_values = ['auto', 's', 'ms', 'mus', 'samples']
+            self._y_values = ['s', 'ms', 'mus', 'samples', 'auto', 'other']
```
Comment (sikersten) [inline pyfar/plot/_interaction.py:386]: 'auto' is not the first anymore. What is the reason for changing the order here?

**Review by sikersten — APPROVED** [review]
Thanks for the effort! Only two minor comments

File: docs/resources/plot_shortcuts.rst (hunk 1 above)
Comment (f-brinkmann) [reply docs/resources/plot_shortcuts.rst]: Thanks for noting. This is generated in pyfar.plot.utils, which can generate it for printing to the console and printing for sphinx. I made sure both versions are fine and extended the corresponding test.

File: pyfar/plot/_interaction.py (hunk 2 above)
Comment (f-brinkmann) [reply pyfar/plot/_interaction.py]: It's a beer. When we changed the behavior of the `unit` parameter for all plots showing time on any axis, we forgot to change it here. I noted an other copy past mistake, which I corrected anywhere: y_type and y_param should be of the same length.

**Pull request merged**